packages/angular/src/stories/datagrid/data/seed-data.ts

```diff
@@ -0,0 +1,2016 @@
+export const FIRST_NAMES: string[] = [
```

Comment:
nit: do we need to have quite so many samples? I think we could easily have max 100 of each to trim this up.

Reply:
trimmed the list.